db/models.js

```diff
@@ -1,35 +1,36 @@
 // Static methods for database interactions
 class Models {
   static getCampaignById(db, id) {
-    return db.query('SELECT * FROM campaigns WHERE id = ?', [id])
+    return db.query('SELECT * FROM campaigns WHERE id = $1;', [id])
+      .then((result) => result.rows[0])
       .catch((err) => { console.error(err); });
   }
 
-  static getCampaignsByAuthor(db, author) {
-    return db.query('SELECT * FROM campaigns WHERE author = ?', [author])
+  static getCampaignsByUser(db, username) {
+    return db.query('SELECT * FROM campaigns WHERE _user = $1;', [username])
+      .then((result) => result.rows)
       .catch((err) => { console.error(err); });
   }
 
   static addCampaign(db, data) {
-    return db.query('INSERT INTO campaigns (campaign, description, author, currency, pledged, goal, backers, endDate, location, _type) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?);', Object.values(data))
+    return db.query('INSERT INTO campaigns (campaign, description, author, _user, country, pledged, goal, backers, endDate, _type) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10);', Object.values(data))
```

Comment:
Why are these hardcoded instead of dynamic? Is this for dev/testing purposes?

Reply:
Postgres value binding uses numbers, the question marks didn't work for me here unlike MySQL. This is indeed dynamic